pom.xml

```diff
@@ -33,15 +33,15 @@
         <version.arquillian_core>1.7.0.Alpha10</version.arquillian_core>
```

Comment:
Update this to <version.arquillian_core>1.8.0.Final</version.arquillian_core> so we pickup the breakout of the Jakarta EE apis from core.

Reply:
Ah, good catch, forgot about that one!
Fixed now